.github/workflows/hax.yml

```diff
@@ -0,0 +1,34 @@
+name: hax
+
+on:
+  workflow_dispatch:
+  push:
+    branches: ['**']
+  pull_request:
+
+jobs:
+  run-hax:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      packages: read
+
+    env:
+      HAX_IMAGE: ghcr.io/cfm/hax@sha256:9200676dcc475677fb18b8f07ea1da1aeb63680eedec8a78c26a760343b0d6a4
```

Comment:
I've built this from <https://github.com/cryspen/hax/blob/4019185e9c96836c51081519bb6b7e51edee1f4c/README.md?plain=1#L159-L170> just to have something reproducible in CI, but we'll probably want to pin an image that we build and publish ourselves.